src/test/java/org/mongojack/TestDBUpdateSerialization.java

```diff
@@ -153,6 +157,30 @@ public void testSimpleMap() {
         assertThat(coll.findOneById("id").map, equalTo(map));
     }
 
+    // Test to detect presence of issue https://github.com/devbliss/mongojack/issues/98
+    @Test
+    public void testUpdateOfNestedRepeatedAttributeName() {
+        coll2 = getCollection(NestedRepeatedAttributeName.class, String.class);
+
+        Calendar cal = Calendar.getInstance();
+        cal.set(0, 0, 0, 0, 0, 10);
+        Date d1 = cal.getTime();
```

Comment:
What if you just did:

    Date d1 = new Date(10_000l);

It doesn't really matter what the date is and that'd be fewer lines of code, which I think makes the test more readable